test/SlowTests/Issues/RDBC-285.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using FastTests;
+using Orders;
+using Raven.Client.Documents;
+using Tests.Infrastructure;
+using Xunit;
+
+namespace SlowTests.Issues
+{
+    public class RDBC_285 : RavenTestBase
+    {
+        [Fact]
+        public void CanQueryWithBoostEqZero()
+        {
+            using (var store = GetDocumentStore())
+            {
+                store.Maintenance.Send(new CreateSampleDataOperation());
```

Comment:
Test failed. We need to wait for indexing after this I think?

Reply:
Yep